lib/board.rb

```diff
@@ -1,5 +1,7 @@
+require_relative 'square'
+
 class Board
-  attr_reader(:current_player, :board_middle, :turn_count, :max_turns, :board_size, :inOut)
+  attr_reader :current_player, :board_middle, :turn_count, :max_turns, :board_size, :inOut, :squares
   BOARD_ROW = '-'
```

Comment:
These constants fee like concerns of something that needs to display the board, but not the logic of how a board works.